### ISSUE TITLE
Neither Macro nor Settings working

### ISSUE THREAD

**Issue by fraggynator** [PoE-TradeMacro/POE-TradeMacro#317]
Sadly, this macro isn't working for me in any way shape or form.
I can use ItemInfo (the Ctrl-C overlay), everything else won't do anything.

It doesn't even matter if I'm using the x64 or x86 client. Windowed fullscreen, no renaming of the exe. AHK version is 1.1.25.01, TradeMacro release redownloaded (latest), other AHK macros work perfectly.
Running Test_WinHTTPRequest.ahk tells me it's not a connection issue:
https://gyazo.com/895add40f03667dbd9f4fcdbcd6efbb8

Furthermore, trying to change any settings throws me a fun and exciting error message loop:
https://gyazo.com/2da5ba9c2911f3bb732567564296871c

If I then switch the keyboard layout (for some reason that eludes me) from Swiss German to US, I get the same damn error loop:
https://gyazo.com/3b5992ff24c49fc167bc93266d53f5a8

**Comment by fraggynator** [comment]
nvm, I got it to work after fiddling with different and seemingly dumb things.
Apparently, my Bitdefender decided to enact ransomware protection on AHK Unicode 64 without showing it in the report for some reason.
When I switched to execute the ahk with AHK Unicode 32, it showed up. After adding this as trusted application I was even able to switch back to AHK U 64.

The really stupid thing about this issue is, that the macro worked on my laptop with the standalone PoE client, even though the same Bitdefender is installed.
